src/content/pages/account/login/index.tsx

```diff
@@ -111,12 +118,43 @@ const LoginForm = () => {
 
         // Don't continue to login logic if there are errors.
         // We cannot check the useState values, as these will not be updated until the next render.
-        if (usernameInvalid || passwordInvalid) return
+        //if (usernameInvalid || passwordInvalid) return
 
         // TODO: Login logic.
 
+        const asyncPostCall = async () => {
+            try {
+                const response = await fetch('http://localhost:8000/user/login', {
+                 method: 'POST',
+                 headers: {
+                   'Content-Type': 'application/json',
+                   },
+                body: JSON.stringify({
+                    email: username,
+                    password: password
+                    })
+                 });
+                 const data = await response.json();
+                 setJWT(data)
+               } 
+               
+               catch(error) {
+                  console.log(error)
+                } 
+            }
+            
+        asyncPostCall()
+        
+
         // Prevent default form action (which would refresh the page).
         e.preventDefault()
+        console.log(JWT)
+        if(JWT){
+            navigate("/management/profile")
+        }else{
+            console.log("Press login again lmao")
```

Comment:
Can this message be a bit more formal? Also missing some whitespace in the lines around it.

Reply:
and also perhaps give a popup because we can't expect users to go to the console to see what's wrong lmao

Reply:
Sorry, forgot to remove it. It was just for Adam and I's testing purposes. Shouldn't be seen by user at all.